packages/api/test/maintenance.spec.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-env mocha */
```

Comment:
I wonder if we should add some more integration-y tests, to make sure we test the modes are set up for the different verbs?
At the moment we are just testing the middleware itself

Reply:
yes it would be good to have at least one test that checks that the `*` style middlewear are actually being applied correctly, as I we've not used them before.

Reply:
Tracked in a separate [issue](https://github.com/web3-storage/web3.storage/issues/1297)